src/components/organisms/UiMultipleChoices/UiMultipleChoices.stories.ts

```diff
@@ -0,0 +1,308 @@
+import {
+  inject,
+  provide,
+  ref,
+} from 'vue';
+import type {
+  Meta,
+  StoryObj,
+} from '@storybook/vue3';
+import { actions } from '@storybook/addon-actions';
+import deepmerge from 'deepmerge';
+import {
+  useArgTypes,
+  extendEvents,
+} from '@sb/helpers';
+import {
+  UiAlert,
+  UiMultipleChoices,
+  UiSidePanel,
+} from '@index';
+import UiMultipleChoicesItem from '@/components/organisms/UiMultipleChoices/_internal/UiMultipleChoicesItem.vue';
+import type { MultipleChoicesProps } from '@index';
+
+type MultipleChoicesArgsType = MultipleChoicesProps;
+type MultipleChoicesMetaType = Meta<MultipleChoicesArgsType>;
+type MultipleChoicesStoryType = StoryObj<MultipleChoicesArgsType>;
+
+const events = extendEvents([
+  'update:modelValue',
+  'update:invalid',
+]);
+
+const buttonEvents = actions({ onClickInfoButton: 'click:info-button' });
+
+const { argTypes } = useArgTypes(deepmerge(UiMultipleChoices, events));
+
+const meta = {
+  title: 'Organisms/uiMultipleChoicesTS',
+  component: UiMultipleChoices,
+  args: {
+    modelValue: [],
+    items: [
+      {
+        label: 'I have diabetes',
+        textLabelAttrs: { 'data-testid': 'label-text' },
+      },
+      {
+        label: 'I have hypertension',
+        textLabelAttrs: { 'data-testid': 'label-text' },
+      },
+      {
+        label: 'I have high cholesterol',
+        textLabelAttrs: { 'data-testid': 'label-text' },
+      },
+    ],
+    options: [
+      {
+        label: 'Yes',
+        value: 'present',
+      },
+      {
+        label: 'No',
+        value: 'absent',
+      },
+      {
+        label: "Don't know",
+        value: 'unknown',
+      },
+    ],
+    hint: 'Select one answer in each row',
+    touched: false,
+    invalid: true,
+    alertHintAttrs: { 'data-testid': 'alert-hint' },
+  },
+  argTypes: {
+    ...argTypes,
+    modelValue: { control: 'object' },
+    items: { control: 'object' },
+    options: { control: 'object' },
+  },
+  decorators: [ () => ({
```

Comment:
? Why we need it? IMO in SidePanel we use that pattern because trigger button be provided by decorator. Here we doesn't need it. We can define value in story